datalab/datalab_session/serializers.py

```diff
@@ -27,3 +27,16 @@ class DataSessionSerializer(serializers.ModelSerializer):
     class Meta:
         model = DataSession
         fields = '__all__'
+
+    def update(self, instance, validated_data):
+      input_data = validated_data.get('input_data', [])
+      
+			# Check for duplicates
```

Comment:
is there some spacing thing going on here?

Reply:
Its spaced properly in my IDE so I'm not sure why its tabbed in here